js/src/main.js

```diff
@@ -566,6 +566,34 @@ import ICONS from './creatives/icons';
     }
     isUrl ? _makeAjaxRequest.call(this, vastUrl) : _parseXml.call(this, vastUrl);
   };
+
+  RmpVast.prototype.pnVideoPlay = function () {
+    this.manualPlay = true;
+
+    if (DEBUG) {
+      FW.log('---- pnVideoPlay called');
+      FW.log('this.vastPlayer.readyState - ' + this.vastPlayer.readyState);
+      FW.log('this.contentPlayer.readyState - ' + this.contentPlayer.readyState);
+    }
+
+    if (this.vastPlayer.readyState > 3 || this.contentPlayer.readyState > 3) {
```

Comment:
can either player be undefined?

Reply:
@solitaa what about this one?

Reply:
I tested few scenarios, like when vast is broken or video is unavailable, rmp works fine and no issue with players, but I agree that it will be more accurate to add checks.